libgalois/src/analytics/louvain_clustering/louvain_clustering.cpp

```diff
@@ -593,15 +623,32 @@ katana::analytics::LouvainClustering(
         pg, temporary_edge_property.name(), output_property_name, plan);
   }
 
+  TemporaryPropertyGuard temporary_edge_property{pg->EdgeMutablePropertyView()};
```

Comment:
I don't think we should add another temp edge_property to the original graph. Please convert the edge-property to int64 when creating duplicate graph to work on.